ddp-workspace/projects/ddp-pancan/src/app/components/faq/faq.component.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../../styles/common.scss';
+
+.faq {
+    padding: 4.56rem 1.5rem 6.12rem 1.5rem;
+    text-align: center;
+
+    .title {
+        @extend .static-page-title;
+        margin-top: 0;
+        margin-bottom: 4.56rem;
+    }
+
+    .text {
+        @extend .static-page-paragraph-text
+    }
+
+    .expand-btn {
+        color: #282828;
+        font-family: Untitled-Sans-Medium;
+        font-size: 1rem;
+        letter-spacing: 0;
```

Comment:
isn't it default value?

Reply:
Removed at 7c8055f26248f550dfcdc8e9260f6caf1ca2a76d